test/filters/test_experimental_filters.cpp

```diff
@@ -0,0 +1,522 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2021-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#include <pcl/common/eigen.h>
+#include <pcl/filters/experimental/crop_box.h>
+#include <pcl/filters/experimental/passthrough.h>
+#include <pcl/io/pcd_io.h> // for loadPCDFile
+#include <pcl/test/gtest.h>
+#include <pcl/pcl_base.h> // for pcl::Indices
+#include <pcl/pcl_tests.h>
+#include <pcl/point_types.h>
+
+using namespace pcl;
+using namespace Eigen;
+
+PointCloud<PointXYZ>::Ptr cloud(new PointCloud<PointXYZ>);
+
+TEST(ExperimentalCropBox, Filters)
+{
+  // Create cloud with center point and corner points
+  PointCloud<PointXYZ>::Ptr input(new PointCloud<PointXYZ>());
+
+  input->push_back(PointXYZ(0.0f, 0.0f, 0.0f));
+  input->push_back(PointXYZ(0.9f, 0.9f, 0.9f));
+  input->push_back(PointXYZ(0.9f, 0.9f, -0.9f));
+  input->push_back(PointXYZ(0.9f, -0.9f, 0.9f));
+  input->push_back(PointXYZ(-0.9f, 0.9f, 0.9f));
+  input->push_back(PointXYZ(0.9f, -0.9f, -0.9f));
+  input->push_back(PointXYZ(-0.9f, -0.9f, 0.9f));
+  input->push_back(PointXYZ(-0.9f, 0.9f, -0.9f));
+  input->push_back(PointXYZ(-0.9f, -0.9f, -0.9f));
+
+  // Create indices vector ( without 0 and 4)
+  pcl::IndicesPtr idx(new pcl::Indices(7));
+  (*idx)[0] = 1;
+  (*idx)[1] = 2;
+  (*idx)[2] = 3;
+  (*idx)[3] = 5;
+  (*idx)[4] = 6;
+  (*idx)[5] = 7;
+  (*idx)[6] = 8;
```

Comment:
Would this work?
```suggestion
  pcl::IndicesPtr idx(new pcl::Indices({1,2,3,5,6,7,8}));
```

Reply:
Should I also do it to the original test?